File: tests/rust/tests-rust-testing-reqwest-selenium/tests/ui/home_layout_tests.rs
use allure_cargotest::allure_test;
use tests_rust_testing_reqwest_selenium as tests;

#[allure_test(name = "Home shows embedded header and reference layout")]
#[tokio::test]
async fn home_layout_is_mounted() {
    tests::layer_ui("Home layout mount", "Home", "Home layout", "normal");
    allure_rust_commons::tag("mock");
    tests::with_browser(|| async {
        tests::HomePage::default()
            .open_page()
            .await
            .should_show_layout()
            .await
            .header
            .should_show_embedded_header()
            .await;
    })
    .await;
}

File: tests/csharp/tests-csharp-xunit-api_request-playwright/Tests/Ui/HomeLayoutTests.cs
using Allure.Net.Commons;
using Allure.Net.Commons.Attributes;
using Tests;

namespace Tests.Ui;

[AllureLabel("layer", "ui")]
[AllureEpic("Home")]
[AllureFeature("Home layout")]
[AllureSeverity(SeverityLevel.normal)]
[AllureSuite("Home layout mount")]
public sealed class HomeLayoutTests : TestBase
{
    [Trait("TestCategory", "ui")]
    [Trait("TestCategory", "mock")]
    [Fact(DisplayName = "Home shows embedded header and reference layout")]
    public void HomeLayoutIsMounted()
    {
        HomePage.OpenPage()
            .ShouldShowLayout()
            .Header.ShouldShowEmbeddedHeader();
    }
}
